### PR TITLE
Vegasmorph patch 1

### PR DESCRIPTION
**Pull request by vegasmorph**
Screenshot_20220610_134012_com android chrome

### CODE REVIEW

**Comment by taransalh1** [comment]
Need to get his merged to master asap .community wants it

File: Tax 1.2% in all transactions
Comment (benjamin-koenig) [inline Tax 1.2% in all transactions:3]: These `“` should be replaced with `"`.

**Review by benjamin-koenig — CHANGES_REQUESTED** [review]

**Comment by benjamin-koenig** [comment]
I am not a Terra dev and don't know the processes but I wonder why you have created a second PR instead of adapting the original one?

File: Tax 1.2% in all transactions
Comment (taransalh1) [reply Tax 1.2% in all transactions:3]: I think its the way you see on git 
Its ok

**Comment by vegasmorph** [comment]
Hi tks for the support... 2 teams of Devs are writing a possible code to be
implemented. That code that you are trying to merge was rushed ,was basacly
a backup plan,before the votation finished.this being sat ,the most
important thing is not even the code but the proposal itself. We need to
create pressure on the DK and terra team to honour the will of the
community.

**Comment by taransalh1** [comment]
What do u think the next steps are ?

**Comment by DoKwonTerra** [comment]
All seems good, implementing now.

**Comment by 1258MX** [comment]
Joker wound up

**Review by DoKwonTerra — APPROVED** [review]
Let's implement the 1.2% tax

**Comment by qasimsaleem1** [comment]
Implement 1.2% transaction fee on luna 2.0 also. And that amount needs to go to burn for a time being at least till 1 year until this all biased dust settles against you this help to support prices.

**Comment by taransalh1** [comment]
Right until we go the specified supply 
What is the burn address we are considering?

**Comment by qasimsaleem1** [comment]
1)Mint other coins on other chains swap it and buy back luna 2.0. once prices settle to $25 to $30. All those criticism will vanished and everyone will be happy to work with you. This will increase market confidence and more projects will launch on Terra ecosystem or join it. 
2)Any project migrate from other chains to terra will be rewarded with funding's. (As we have extreme good amount of funding in emergency luna allocation)
3) Luna 2.0 burning from transaction fees
By any means settle the price to $25 to $30 otherwise you and your developers or from your circle will be in trouble in the days ahead. After 6 months vested coins will be released which will drag the prices down. If all these three things are implemented the coin release will not affect the prices much as the market will have the capacity to absorb it. @DoKwonTerra

**Comment by DoKwonTerra** [comment]
First of all, i would like to apologize for the disaster that happened to LUNA & UST which is affecting to all of the Lunatics. By implementing this 1.2% tax burn rate, i hope all of you could forgive me.

**Comment by 1258MX** [comment]
@DoKwonTerra Facke accaunt

**Comment by qasimsaleem1** [comment]
All people amounts are in 2.0 vested and prices are down and many new investors have invested in luna 2.0 hoping this will settle. If prices remain down for long believe me more and more things or disaster will happen like JAIL for whole life. You will spent life without your loved ones. Use your brain and help investor recoup some of losses into profit by moving prices to $25 to $30 for luna 2.0

**Comment by taransalh1** [comment]
the thing thats making me crazy is the reference to something on top that says 
"Approved the changes"
this means its merged isnt it?

**Comment by ZaradarBH** [comment]
As I understand it the proper value for a 1.2% tax increase is 0.012000000000000000 (sdk.NewDecWithPrec(12, 3)) and not 1.200000000000000000, which == 120% or (sdk.NewDecWithPrec(12, 1)). 

Furthermore the tax rate is modified by the TreasuryKeeper each epoch according to this doc: https://github.com/terra-rebels/classic-core/blob/main/x/treasury/spec/01_concepts.md#updating-policies, so simply applying the current proposal will not maintain a constant burn tax over time as the mining incentives are added each epoch?

**Comment by taransalh861** [comment]
@DoKwonTerra looks to be Fake account but how did he approved the request??

**Comment by ZaradarBH** [comment]
Anyone can approve a PR m8. Merging is reserved to people with org perms :)

**Comment by luncbrasil** [comment]
> @DoKwonTerraparece ser conta Fake mas como ele aprovou o pedido??

to suspeitando também que seja fake, tomare que não tomara que seja o mesmo

**Comment by luncbrasil** [comment]
> Pelo que entendi, o valor adequado para um aumento de imposto de 1,2% é 0,012000000000000000 (sdk.NewDecWithPrec(12, 3)) e não 1,200000000000000000, que == 120% ou (sdk.NewDecWithPrec(12, 1)).
> 
> Além disso, a taxa de imposto é modificada pelo TreasuryKeeper a cada época de acordo com este documento: https://github.com/terra-rebels/classic-core/blob/main/x/treasury/spec/01_concepts.md#updating-policies , então a simples aplicação da proposta atual não manterá um imposto de queima constante ao longo do tempo, pois os incentivos à mineração são adicionados a cada época?
olha verifiquem isso, tambem to achando que vão colocar como 120% , concordo com colega acredito que na codificação 1,2% = 0,012

**Comment by luncbrasil** [comment]
Gente lembre-se, isso é lidar com vidas financeiras a nivel global. se atentem por favor

**Comment by gamediy8** [comment]
> Screenshot_20220610_134012_com android chrome

**Comment by penniescoin** [comment]
Any news?The 1.2% transaction tax needs to be added urgently. Proposal 3568 should be implemented ASAP.Thanks.

**Comment by engfmaia** [comment]
@DoKwonTerra @dokwon time to approve or reject this PR. The community has spoken. In case you totally do not care about Luna classic future, is time for you to step down and give the write access to the community

**Comment by fullyconnected** [comment]
> @DoKwonTerra @dokwon time to approve or reject this PR. The community has spoken. In case you totally do not care about Luna classic future, is time for you to step down and give the write access to the community

DoKwonTerra is a fake account. Please don't include him any requests.

**Comment by fullyconnected** [comment]
image

**Comment by fullyconnected** [comment]
We are awaiting approval for this proposal.

**Comment by 1258MX** [comment]
Approval may not happen in this era. It remains only to wait for Terra Labs. When this will happen is unknown. (

**Comment by Bakartan** [comment]
@Jared-TFL can you merge this proposal, please?

**Comment by littlekira01** [comment]
you have to add a module for the parameter and a recipient to the code for the burn process, without it will not be possible to recognize the code. from the page: https://docs.terra.money/docs/develop/terra-js/add-modules.html

**Comment by 1258MX** [comment]
How far has the process progressed ?

**Pull request closed without merging**
